<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsObjHabilidadesWidget</class>
 <widget class="QWidget" name="ConfigsObjHabilidadesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxHabilidades">
     <property name="title">
      <string>Habilidades</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="botaoAddHabilidade">
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonEdit">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonRemove">
        <property name="text">
         <string>Remover</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QTreeWidget" name="treeWidgetSkills">
        <column>
         <property name="text">
          <string>Habilidade</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Intensidade</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Level</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sons</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ao levar dano:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditDamageSound">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButtonDamageSound">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButtonRemoveDamageSound">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSpecialSound">
        <property name="text">
         <string>Som especial:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSpecialSound">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButtonSpecialSound">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="toolButtonRemoveSpecialSound">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxEventos">
     <property name="title">
      <string>Eventos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="botaoEditorEventos">
        <property name="text">
         <string>Ir para o editor de eventos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
